Vault-Deployment/values/kube-prometheus-stack-values.yaml

```diff
@@ -854,11 +854,11 @@ grafana:
         # datasourceUid: Jaeger
         # traceIdLabelName: trace_id
 
-  extraConfigmapMounts: []
-  # - name: certs-configmap
-  #   mountPath: /etc/grafana/ssl/
-  #   configMap: certs-configmap
-  #   readOnly: true
+  extraConfigmapMounts:
```

Comment:
Very important to mount the correct "rule" file/YAML to the correct service. Here we mount the promql alerts to the Grafana pod so we don't need to configure the alerts manually in Grafana UI.